store/store.go

```diff
@@ -317,46 +327,24 @@ func init() {
 		panic(err)
 	}
 
-	defaultConfig.SearchURI, err = storeBaseURI.Parse("api/v1/snaps/search")
-	if err != nil {
-		panic(err)
-	}
-
-	// slash at the end because snap name is appended to this with .Parse(snapName)
-	defaultConfig.DetailsURI, err = storeBaseURI.Parse("api/v1/snaps/details/")
-	if err != nil {
-		panic(err)
-	}
-
-	defaultConfig.BulkURI, err = storeBaseURI.Parse("api/v1/snaps/metadata")
+	assertsBaseURI, err := url.Parse(assertsURL(storeBaseURI))
 	if err != nil {
 		panic(err)
 	}
 
-	assertsBaseURI, err := url.Parse(assertsURL())
+	myappsBaseURI, err := url.Parse(myappsURL())
 	if err != nil {
 		panic(err)
 	}
 
-	defaultConfig.AssertionsURI, err = assertsBaseURI.Parse("assertions/")
-	if err != nil {
-		panic(err)
-	}
-
-	defaultConfig.OrdersURI, err = url.Parse(myappsURL() + "purchases/v1/orders")
-	if err != nil {
-		panic(err)
-	}
-
-	defaultConfig.CustomersMeURI, err = url.Parse(myappsURL() + "purchases/v1/customers/me")
-	if err != nil {
-		panic(err)
-	}
-
-	defaultConfig.SectionsURI, err = storeBaseURI.Parse("api/v1/snaps/sections")
-	if err != nil {
-		panic(err)
-	}
+	defaultConfig.SearchURI = urlJoin(storeBaseURI, "api/v1/snaps/search")
+	// slash at the end because snap name is appended to this with .Parse(snapName)
+	defaultConfig.DetailsURI = urlJoin(storeBaseURI, "api/v1/snaps/details/")
+	defaultConfig.BulkURI = urlJoin(storeBaseURI, "api/v1/snaps/metadata")
+	defaultConfig.AssertionsURI = urlJoin(assertsBaseURI, "assertions/")
+	defaultConfig.OrdersURI = urlJoin(myappsBaseURI, "purchases/v1/orders")
+	defaultConfig.CustomersMeURI = urlJoin(myappsBaseURI, "purchases/v1/customers/me")
+	defaultConfig.SectionsURI = urlJoin(storeBaseURI, "api/v1/snaps/sections")
```

Comment:
feels like this block of code should be reorderd  by first storeBaseURI stuff, then assertionsBaseURI, then the myappsBaseURI left overs